agent/functional_tests/tests/functionaltests_unix_test.go

```diff
@@ -387,6 +387,12 @@ func TestTelemetryWithStatsPolling(t *testing.T) {
 	telemetryTestWithStatsPolling(t, "telemetry")
 }
 
+// TestTelemetryStorageStats tests whether agent can send metrics to TACS,
+// via cloudwatch metrics.  This is an end-to-end test.
+func TestTelemetryStorageStats(t *testing.T) {
```

Comment:
should the test be added for windows as well?

Reply:
cloudwatch insights is not supported on windows yet.  There's a separate issue tracking this.